src/main.rs

```diff
@@ -110,9 +84,18 @@ package() {{
 }
 
 fn main() {
-    generate_pkgbuild(
-        &read_manifest(&get_manifest_path()).unwrap(),
-        &mut File::create("PKGBUILD").unwrap(),
-    )
-    .unwrap();
+    fn exec() -> Result<()> {
+        let manifest_path = locate_cargo_manifest::locate_manifest()
+            .with_context(|| anyhow!("Failed to locate `{MANIFEST_FILENAME}`"))?;
+        let manifest = parse_manifest(&manifest_path)
+            .with_context(|| anyhow!("Failed to parse `{}`", manifest_path.display()))?;
+        let mut pkgbuild = File::create("PKGBUILD").context("Failed to create PKGBUILD file")?;
+        generate_pkgbuild(&manifest, &mut pkgbuild).context("Failed to generate PKGBUILD")?;
+        Ok(())
+    }
```

Comment:
Why not just make `fn main() -> Result<T>` and return from it directly?

Reply:
You are right, this does have the same behavior. Fixed in f30bb6168e8e313f3b820f8d43282fef58438f8d